components/InputFormGuest.js

```diff
@@ -11,11 +11,57 @@ export default function AddGuest() {
           <Input
             className="SmallFontStyle"
             type="text"
-            id="contactData"
-            name="contactData"
+            id="firstName"
+            name="firstName"
             required
+            placeholder="Vorname"
+          />
+          <Input
+            className="SmallFontStyle"
+            type="text"
+            id="lastName"
+            name="lastName"
+            required
+            placeholder="Nachname"
+          />
+          <Input
+            className="SmallFontStyle"
+            type="text"
+            id="address"
+            name="address"
+            required
+            placeholder="Anschrift"
+          />
+          <label className="MediumFontStyle" htmlFor="companion">
+            Begleitpersonen
+          </label>
```

Comment:
There is no corresponding input element for this label, is it?

Reply:
Yes, there is no input field. I´ll change that.